lib/src/screens/supa_phone_auth.dart

```diff
@@ -33,6 +38,7 @@ class _SupaPhoneAuthState extends State<SupaPhoneAuth> {
 
   @override
   Widget build(BuildContext context) {
+    final signingIn = widget.phoneAuthAction == PhoneAuthAction.signIn;
```

Comment:
Very minor thing, but if it's a boolean, I like to start the variable name with `is`
```suggestion
    final isSigningIn = widget.phoneAuthAction == PhoneAuthAction.signIn;
```